app/views/businessmatching/summary.scala.html

```diff
@@ -89,17 +89,31 @@
             editLinkTag = "businessactivities-edit"
         ) {
             <ul class="list">
-                @activity.businessActivities.toList.sortBy(_.toString).map { line =>
-                    <li>@line match {
-                        case AccountancyServices => {@Messages("businessmatching.registerservices.servicename.lbl.01")}
-                        case BillPaymentServices => {@Messages("businessmatching.registerservices.servicename.lbl.02")}
-                        case EstateAgentBusinessService => {@Messages("businessmatching.registerservices.servicename.lbl.03")}
-                        case HighValueDealing => {@Messages("businessmatching.registerservices.servicename.lbl.04")}
-                        case MoneyServiceBusiness => {@Messages("businessmatching.registerservices.servicename.lbl.05")}
-                        case TelephonePaymentService => {@Messages("businessmatching.registerservices.servicename.lbl.07")}
-                        case TrustAndCompanyServices => {@Messages("businessmatching.registerservices.servicename.lbl.06")}
-                    }</li>
-                }
+            @activity.businessActivities.toList.sortBy(_.getMessage()).map { line =>
```

Comment:
@matthewharris1 - this is all Slavs work that has been tested; approved and merged to master. This PR is simply to get mater into auth-wip to keep the auth branch up to date to avoid merge conflicts. 

Sorry; I should have been more clear. I do not want to refactor master in auth-wip.

Reply:
Thats my bad. Looks good to merge in that case

Reply:
Cheers :-)